src/control/lib/control/storage.go

```diff
@@ -240,7 +239,7 @@ func StorageScan(ctx context.Context, rpcClient UnaryInvoker, req *StorageScanRe
 				Basic: req.NvmeBasic,
 				// Health and meta details required to populate usage statistics.
 				Health: req.NvmeHealth || req.Usage,
-				Meta:   req.NvmeMeta || req.Usage,
+				Meta:   req.Usage,
```

Comment:
Do we still need to for compatibility reason with the engine.
If yes, is it planned to also remove it from the engine ?

Reply:
where should it be removed in the engine?